File: webclient/src/app/models/address.ts
export class Address
{
    private country:string;
    private city:string;
    private town:string;

    public constructor(country:string,city:string,town:string)
    {
        this.country = country;
        this.city = city;
        this.town = town;
    }

    
    public get getCountry(): string {
        return this.country;
    }

    public set setCountry(value: string) {
        this.country = value;
    }

    public get getCity(): string {
        return this.city;
    }
    
    public set setCity(value: string) {
        this.city = value;
    }

    public get getTown(): string {
        return this.town;
    }
    
    public set setT123123123own(value: string) {
        this.town = value;
    }
}

File: src/Address.java
public class Address {
    private String country;
    private String city;
    private String town;

    public Address(String country, String city, String town) {
        this.country = country;
        this.city = city;
        this.town = town;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getTown() {
        return town;
    }

    public void setTown(String town) {
        this.town = town;
    }

    @Override
    public String toString() {
        return town+","+city+","+country;

    }



}